### PR TITLE
refactor: improved readability

### PR DESCRIPTION
**Pull request by thinkrapido**

### CODE REVIEW

**Comment by tijlleenders** [comment]
@thinkrapido 
Just curious: why does improved readabilty require nightly? 

Is it the 'with-logging' feature?  
Where can I find docs for that? I'd like to understand what it does.

**Comment by thinkrapido** [comment]
> @thinkrapido Just curious: why does improved readabilty require nightly?
> 
> Is it the 'with-logging' feature? Where can I find docs for that? I'd like to understand what it does.

https://github.com/rust-lang/rustfmt

blank_lines_lower_bound for instance is only on nightly toolchain recognized

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+root = true
+
+[*]
+trim_trainling_whitespace = true
```
Comment (kobe-reygel) [inline .editorconfig:4]: Should be trim_trailing_whitespace I suppose?

**Comment by kobe-reygel** [comment]
@tijlleenders our rust build for the tests in the pipeline also already runs on nightly. The logic behind this (if I understand it correctly) is catching things early: preventing us from merging stuff that will be regarded as 'wrong' in the next stable build.

I therefore think that Romeo's suggestion of also using it for rustfmt is a good one

File: Cargo.toml [hunk 2]
```diff
@@ -13,6 +13,7 @@ categories = ["algorithms", "date-and-time", "wasm"]
 [features]
 experimental-testset = []
 skip-test-generation = []
+with-logging = []
```
Comment (kobe-reygel) [inline Cargo.toml:16]: Great idea

**Review by kobe-reygel — APPROVED** [review]
Looks good

**Pull request merged**